From 51656a9021096d791f6bf32905b98a8a0b9a2d7e Mon Sep 17 00:00:00 2001
From: Garrett Brown <eigendebugger@gmail.com>
Date: Fri, 12 Nov 2021 22:00:37 +0000
Subject: [PATCH] Change rcpputils to "master" branch

Needed to fix error:

  camera_info_manager.cpp:46:10: fatal error: rcpputils/env.hpp: No such file or directory
     46 | #include "rcpputils/env.hpp"
        |          ^~~~~~~~~~~~~~~~~~~
---
 ros2.repos | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ros2.repos b/ros2.repos
index e9643c6..8c9e162 100644
--- a/ros2.repos
+++ b/ros2.repos
@@ -278,7 +278,7 @@ repositories:
   ros2/rcpputils:
     type: git
     url: https://github.com/ros2/rcpputils.git
-    version: galactic
+    version: master
   ros2/rcutils:
     type: git
     url: https://github.com/ros2/rcutils.git
-- 
2.25.1

